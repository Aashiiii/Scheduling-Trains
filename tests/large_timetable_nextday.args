Allawah Oatley "02:44:51"
