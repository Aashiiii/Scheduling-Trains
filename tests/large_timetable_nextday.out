The next train to Oatley from Allawah departs at 02:44:55
